qualtran/bloqs/arithmetic/multiplication.py

```diff
@@ -130,6 +137,29 @@ def build_call_graph(self, ssa: 'SympySymbolAllocator') -> Set['BloqCountT']:
         num_toff = self.bitsize * (self.bitsize - 1)
         return {(Toffoli(), num_toff)}
 
+    def add_my_tensors(
```

Comment:
Add unit tests for both?

Reply:
done. Also identified a bug and fixed it as part of adding the tests